v2g-liberty/rootfs/root/appdaemon/apps/v2g_liberty/modbus_evse_client.py

```diff
@@ -143,6 +157,12 @@ class ModbusEVSEclient:
         "ha_entity_name": "serial_number_low",
     }
 
+    # Groups of entities for efficient reading of the modbus registers.
+    CHARGER_POLLING_ENTITIES: list
+    CHARGER_ERROR_ENTITIES: list
+    # To be read once at init
```

Comment:
Suggestion: instead of suggesting a usage in the comment, would it not make more sense to state that these entities contain "static" data, that only needs to be initialised once?
When exactly that happens seems like an implementation detail that does not belong in a comment.

Reply:
👍